patches/server/1038-Fix-UnsafeValues-loadAdvancement.patch

```diff
@@ -22,7 +22,17 @@ index 96f6e0554baf5915dd1f5b93f3bcfe7a13393c29..806fb1064a1769d1251f2a2b04372275
 +            mapBuilder.put(minecraftkey, holder);
 +
 +            MinecraftServer.getServer().getAdvancements().advancements = mapBuilder.build();
-+            MinecraftServer.getServer().getAdvancements().tree().addAll(List.of(holder));
++            final net.minecraft.advancements.AdvancementTree tree = MinecraftServer.getServer().getAdvancements().tree();
++            tree.addAll(List.of(holder));
++
++            // recalculate advancement position
++            final net.minecraft.advancements.AdvancementNode node = tree.get(minecraftkey);
++            if (node != null) {
++                final net.minecraft.advancements.AdvancementNode root = net.minecraft.advancements.AdvancementNode.getRoot(node);
```

Comment:
You can simplify this by using node.root()

Reply:
Good point. I've updated the patch.